src/sentry/web/frontend/create_organization_member.py

```diff
@@ -78,7 +78,7 @@ def handle(self, request, organization):
                 (r, r in allowed_roles)
                 for r in roles.get_all()
             ],
-            'all_teams': all_teams
+            'all_teams': list(all_teams),
```

Comment:
This just saves a query or two since it coerces the queryset into being evaluated, so the `|length` checks in the template don't add unnecessary queries.